scrapers/vic/yarra.py

```diff
@@ -17,29 +18,51 @@ class YarraScraper(BaseScraper):
     def __init__(self):
         council = "yarra"
         state = "VIC"
-        base_url = ""
+        base_url = "https://www.yarracity.vic.gov.au"
         super().__init__(council, state, base_url)
 
     def scraper(self) -> ScraperReturn | None:
-        # self.logger.info(f"Starting {self.council_name} scraper")
-
-        self.logger.error(f"{self.council_name} is without a Scraper, Can you Help!")
-        return None
-        """
-        YOUR CODE HERE
-        """
-
-        # scraper_return = ScraperReturn(name, date, time, self.base_url, download_url)
-
-        # self.logger.info(f"""
-        #     {scraper_return.name}
-        #     {scraper_return.date}
-        #     {scraper_return.time}
-        #     {scraper_return.webpage_url}
-        #     {scraper_return.download_url}"""
-        # )
-        # self.logger.info(f"{self.council_name} scraper finished successfully")
-        # return scraper_return
+        self.logger.info(f"Starting {self.council_name} scraper")
+        initial_webpage_url = "https://www.yarracity.vic.gov.au/about-us/council-and-committee-meetings/upcoming-council-and-committee-meetings"
+
+        output = self.fetch_with_requests(initial_webpage_url)
+        output = output.content
+
+
+        name = None
+        date = None
+        time = None
```

Comment:
Extract time into a standard format is desired even if not needed currently. Can be regex found or if the website is consistent use a css selector to locate. E.g.

```
#content > div > div > div.card.card-1-4 > div > div:nth-child(3) > p:nth-child(2) > br:nth-child(2)
```

Reply:
completely missed the time on the website, will fix